server/server/src/serve_frontend_plugins.rs

```diff
@@ -0,0 +1,42 @@
+use actix_web::{
+    error, get,
+    web::{self, Data},
+    Error, HttpResponse,
+};
+
+use repository::RepositoryError;
+use service::{
+    plugin::{FrontendPluginFileRequest, FrontendPluginFileRequestError},
+    service_provider::ServiceProvider,
+};
+
+pub fn config_server_frontend_plugins(cfg: &mut web::ServiceConfig) {
+    cfg.service(serve);
+}
+
+#[derive(thiserror::Error, Debug)]
+#[error(transparent)]
+struct DatabaseError(RepositoryError);
```

Comment:
Wrapping an error here so can auto implement ResponseError (for foreign type) and then can easily map_err(Error) below